types/jws/index.d.ts

```diff
@@ -143,10 +143,14 @@ export interface VerifyOptions {
     encoding?: string|Buffer|stream.Readable;
 }
 
-export type Algorithm = 'HS256' | 'HS384' | 'HS512' | 'RS256' |
-                        'RS384' | 'RS512' | 'ES256' | 'ES384' |
-                        'ES512' | 'PS256' | 'PS384' | 'PS512' |
-                        'none';
+export const ALGORITHMS: [
```

Comment:
I don't know that having these separated into two different exported properties provides much value.  Adding the new algorithm types is nice though!

Reply:
@JustinBeckwith, is this an approval?

Reply:
@JustinBeckwith one is a type, the other one is a value that is preserved at runtime. Can be used for things like config validation. Regardless, this is something the author of the library chose to export. Only makes sense that the typings match that.